snitun/server/listener_peer.py

```diff
@@ -58,7 +61,14 @@ def __init__(self, peer_manager: PeerManager, host=None, port=None):
 
             # Start multiplexer
             await peer.init_multiplexer_challenge(reader, writer)
-            await peer.wait_disconnect()
+
+            while peer.is_connected:
+                try:
+                    async with async_timeout.timeout(14400):
```

Comment:
Let's extract this in a constant. This is also 4 hours ?!

Reply:
Oh that's when we will disconnect a client, forcing to reconnect.

Reply:
Note that we will have to redistribute new custom components if we deploy this, because we don't reconnect yet right?

Reply:
done for tests :)